lib/util.c

```diff
@@ -1779,14 +1779,20 @@ grn_path_copy(grn_ctx *ctx,
   }
 
   int destination_fd;
+  errno = 0;
   grn_open(destination_fd,
            destination_path,
            O_WRONLY | O_CREAT | O_EXCL | GRN_OPEN_FLAG_BINARY);
```

Comment:
```suggestion
           O_WRONLY | O_CREAT | O_TRUNC | GRN_OPEN_FLAG_BINARY);
```

Is overwriting better?